tests/x11/shutdown.pm

```diff
@@ -20,6 +20,13 @@ use utils;
 
 sub run() {
     my $self = shift;
+    x11_start_program('xterm');
+    script_sudo(q{echo 'ForwardToConsole=yes' >> /etc/systemd/journald.conf});
+    script_sudo(q{echo 'MaxLevelConsole=debug' >> /etc/systemd/journald.conf});
+    script_sudo(qq{echo 'TTYPath=/dev/$serialdev' >> /etc/systemd/journald.conf});
+    script_sudo(q{systemctl restart systemd-journald});
```

Comment:
don't we have this enabled by the boot parameters already or only during installation? It would be good if you could verify extra_tests_in_textmode after this work, still.